tests/nightwatch/specs/current/contact-page.js

```diff
@@ -7,8 +7,14 @@ function text(n) {
 function contactPage(client) {
   client
     .startService()
-    .click('#callback-link')
-  ;
+    .click('#callback-link', function() {
+      console.log('     ◇ "Get in touch" link clicked');
+    })
+    .waitForElementVisible('.reasons-for-contacting-form', 3000,
+      '  - "Reasons for contacting" form exists')
+    .submitForm('.reasons-for-contacting-form', function() {
+      console.log('     ◇ "Continue to contact CLA" button clicked');
```

Comment:
Can I suggest using `⟡` as I updated that character in [another PR](https://github.com/ministryofjustice/cla_public/pull/496/files#diff-7ce2dbe1781eb931b65932e83ea02e0eR22). I think it works better with other terminal fonts.